config/environments/development.rb

```diff
@@ -97,7 +97,7 @@
   # config.action_view.raise_on_missing_translations = true
 
   # Devise requires a default URL
-  config.action_mailer.default_url_options = { host: ENV['FQDN'], port: ENV['PORT'] }
+  config.action_mailer.default_url_options ||= {}
```

Comment:
Why are we removing action_mailer defaults?  Do they inherit from `route.default_url_options`?

Reply:
Oops, this was a mistake. Fixed

Reply:
So you're removing them from action_mailer.  I thought we needed those for mailhog to work.

Reply:
Oops, this config should be untouched in this PR, I was just failing to leave it alone 🤦‍♂️